src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.ts

```diff
@@ -45,9 +52,50 @@ export class CreatePhotoFormComponent implements OnInit {
   }
 
   private activateEditMode(): void {
-  this.PhotoFormGroup.patchValue(this.provider, { emitEvent: false });
-  this.provider.institutionStatusId = this.provider.institutionStatusId || Constants.SOCIAL_GROUP_ID_ABSENT_VALUE;
-}
+    this.PhotoFormGroup.patchValue(this.provider, { emitEvent: false });
+    this.provider.institutionStatusId = this.provider.institutionStatusId || Constants.SOCIAL_GROUP_ID_ABSENT_VALUE;
+    if (this.provider.providerSectionItems?.length) {
+      this.provider.providerSectionItems.forEach((item: ProviderSectionItem) => this.SectionItemsFormArray.push(this.newForm(item)))
+    } else {
+      this.onAddForm();
+    }
+  }
+
+  /**
+ * This method creates new FormGroup
+ */
+  private newForm(item?: ProviderSectionItem): FormGroup {
+    const EditFormGroup = this.formBuilder.group({
+      name: new FormControl('', [Validators.required]),
+      description: new FormControl('', [
+        Validators.required,
+        Validators.minLength(ValidationConstants.INPUT_LENGTH_1),
+        Validators.maxLength(ValidationConstants.MAX_DESCRIPTION_LENGTH_2000)
+      ]),
+    });
+    
+    if (item){
+      EditFormGroup.addControl('providerId', this.formBuilder.control(item.providerId));
```

Comment:
add please this control to the form, but patch value in constructor if it is passed (it will help with dirty form in the future) Check this flow in this PR https://github.com/ita-social-projects/OoS-Frontend/pull/1308